Documentation/proposals/OneDotNetEmbeddedResources.md

```diff
@@ -1,216 +1,7 @@
 # [proposal] Embedded Resources and `.nupkg` Files
```

Comment:
I think this file can be *removed*; that appears to be the practice in e.g. dotnet/csharplang, e.g. https://github.com/dotnet/csharplang/commit/c3df20406f43fcd460cfedd1cd61b6cc47d27250#diff-c81858101b134a23eb7a0df15e102f4d (which contains a `git mv` from `/proposals` to `/proposals/csharp-9.0`).